x-pack/solutions/security/plugins/security_solution/public/threat_intelligence/modules/indicators/components/common/field_value.tsx

```diff
@@ -41,7 +58,14 @@ export const IndicatorFieldValue: FC<IndicatorFieldValueProps> = ({ indicator, f
   return fieldType === 'date' ? (
     <DateFormatter date={value as string} />
   ) : value ? (
-    <>{value}</>
+    <div
+      css={css`
+        display: flex;
+        flex-direction: column;
+      `}
+    >
+      {multipleFieldValues}
+    </div>
```

Comment:
I think this change actually broke some functionality. We 're not displaying any values that are not within an array now. So take a look at this video and see that we can't filter by the id anymore

https://github.com/user-attachments/assets/2d236c37-b26d-41c0-8f0d-9acba334d3c3

But if I try to do the same on `main` it works just fine

https://github.com/user-attachments/assets/4b907889-c2b2-412f-914b-86e8a681749c